src/ProjectTemplates/test/Templates.Tests/ApiTemplateTest.cs

```diff
@@ -37,7 +37,7 @@ public async Task ApiTemplateCSharp()
     }
 
     [ConditionalFact]
-    [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/47478", Queues = "OSX.13.Amd64.Open;Ubuntu.2004.Amd64.Open;Windows.11.Amd64.Client.Open;")]
+    [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/47478", Queues = "All.OSX;All.Ubuntu;Windows.11.Amd64.Client.Open;Windows.Amd64.Server2022.Open;windows.11.arm64.open")]
```

Comment:
Just to check

> Windows.11.Amd64.Client.Open;Windows.Amd64.Server2022.Open;windows.11.arm64.open

None of those Helix queues have VS installed on them, right?

Reply:
Should we extract this string into a constant, ex. `NativeAotSupportedHelixQueues` or similar?

Reply:
Amd64.Server2022 doesn't seem to
Windows.11.Arm64 was failing with: Unable to find package Microsoft.AspNetCore.App.Runtime.win-arm64 with version (= 8.0.0-ci) so unsure if it has VS or not
Windows.11.Amd64 was already skipped. Don't know its status.

Reply:
I've just put up a PR for a new attribute to try and make this easier moving forward:

https://github.com/dotnet/aspnetcore/pull/48117

PTAL.